airdrop-machine/sources/airdrop_machine.move

```diff
@@ -29,22 +30,33 @@ module airdrop_machine::airdrop_machine {
     /// Token Minting has not yet started.
     const EMINTING_HAS_NOT_STARTED: u64 = 3;
     
-    struct MetadataConfig has store, copy, drop {
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
```

Comment:
If this is an object, perhaps we should store this resource in the same object signer as CollectionConfig. And when fetching it, use `borrow_global<MetadataConfig>(object::object_address(&collection_config_object);`

We can then remove `store, copy` drop` capabilities too from the struct.

Reply:
after thinking more, i don't see the necessity of having to use objectgroup actually.